src/web/components/Progress/Progress.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Progress from './';
+
+describe('Progress', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<Progress value="50" />);
+  });
+
+  it('should render a progress bar with 0% completed', () => {
+    wrapper.setProps({ value: '0' });
```

Comment:
Why to use a string ?

Reply:
I chose to use a string because it was a percentage and I also found it easier to make value="0" than value={0}.
However if this is not correct, I  will change it.